content/docs/configuration/vault.md

```diff
@@ -260,7 +262,43 @@ needs to talks to Vault.
 Although it is not recommended, you can also use the same Vault role for all of
 your Issuers and ClusterIssuers by omitting the `audience` field and re-using
 the same service account.
-<a name="static-service-account-token"></a>
+
+#### Secretless Authentication with a Service Account (External Vault)
+
+If you are using a Vault instance external to your cluster, you will need to set
+the `audiences` to an andience accepted by your Kubernetes cluster. When using
+an external Vault instance, the short-lived token created by cert-manager to
+authenticate to Vault will be used by Vault for authenticating to Kubernetes.
+First, find what your cluster's issuer is:
+
+```sh
+kubectl get --raw /.well-known/openid-configuration | jq .issuer -r
+```
+
+Then, set the `audiences` field to the issuer URL:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: vault-issuer
+  namespace: sandbox
+spec:
+  vault:
+    path: pki_int/sign/example-dot-com
+    server: https://vault.local
+    auth:
+      kubernetes:
+        role: my-app-1
+        mountPath: /v1/auth/kubernetes
+        serviceAccountRef:
+          name: vault-issuer
+          audiences: [https://kubernetes.default.svc.cluster.local]
```

Comment:
Shouldn't the audiences also include something like vault to tighten the integration?

Reply:
The default "computed" audience (e.g., `vault://namespace/issuer-name`) is still added to the audiences by default, so I'd say it would be redundant to add `vault` there? Of course, that's only useful if folks are also using `audience` in their vault role, otherwise the audience isn't checked by vault.

Reply:
Ah yeah, perhaps I would add a sentence to say that the default behavior will also include `vault://namespace/issuer-name` which you can use in the vault role as well.

Reply:
Done